guides/common/modules/con_troubleshooting-postgresql.adoc

```diff
@@ -0,0 +1,30 @@
+[id="Troubleshooting_Postgresql_{context}"]
+= Troubleshooting PostgreSQL
+
+If the PostgreSQL 10 module has already been enabled, perform a module reset:
```

Comment:
Should we go ahead and update this to `12`?

Reply:
Updated :+1: